odoo/sql_db.py

```diff
@@ -245,9 +256,22 @@ def execute(self, query, params=None, log_exceptions=None):
             encoding = psycopg2.extensions.encodings[self.connection.encoding]
             _logger.debug("query: %s", self._obj.mogrify(query, params).decode(encoding, 'replace'))
         now = time.time()
+        catch_traceback_query = bool(os.environ.get('catch_traceback_query', ''))
         try:
             params = params or None
             res = self._obj.execute(query, params)
+            if catch_traceback_query:
+                for query_to_catch in QUERY_TO_CATCHS:
+                    query = self._obj.query.replace(b"\n", b" ").replace(b"\'", b'"')
```

Comment:
...I mean, here using 
`query = self._obj.query.replace(b"\n", b" ").replace(b"\'", b'"').replace(b' ', b'')`

Reply:
Is the inverse diagonal is needed here?

```suggestion
                    query = self._obj.query.replace(b"\n", b" ").replace(b"'", b'"')
```

Reply:
if I print the raw query when I do the following search, the string return `\'`, so, the inverse diagonal it's not to say that it's a simple `'` it's because I want to remove it from this case
```python
In [3]: self.env["sale.order"].search([("state", "in", ("sale", "done"))], limit=1)
b'SELECT "sale_order".id FROM "sale_order" WHERE ("sale_order"."state" in (\'sale\',\'done\')) ORDER BY "sale_order"."date_order" DESC,"sale_order"."id" DESC  limit 1'
Out[3]: sale.order(857571,)
````

 if i apply the suggested change, the resultant query will be something like

```python
b'SELECT "sale_order".id FROM "sale_order" WHERE ("sale_order"."state" in (\"sale\",\"done\")) ORDER BY "sale_order"."date_order" DESC,"sale_order"."id" DESC  limit 1'
```